external/mdal/frmts/mdal_selafin.cpp

```diff
@@ -399,9 +399,9 @@ void MDAL::SelafinFile::populateDataset( MDAL::Mesh *mesh, std::shared_ptr<MDAL:
   }
 
   // now calculate statistics
-  for ( const std::shared_ptr<DatasetGroup> group : groupsInOrder )
+  for ( const std::shared_ptr<DatasetGroup> &group : groupsInOrder )
```

Comment:
ping @vcloarec

Reply:
what's the ping for? something to worry?

Reply:
To make sure it's applied upstream?

Reply:
Ah, good point. Shouldn't block then?

Reply:
It likely should. I'm sure that'll be applied upstream prior to the next mdal update in QGIS' source tree. In the meantime, good to have it fixed here.

Reply:
Yep, just warning mdal guys to port upstream.

Reply:
Cool, I'll merge then, this will also ensure that any future mdal update comes with this patch applied.

Reply:
ported https://github.com/lutraconsulting/MDAL/pull/289 👍 

but don't be shy, your direct contributions to MDAL are welcomed too!

Reply:
> ported [lutraconsulting/MDAL#289](https://github.com/lutraconsulting/MDAL/pull/289) +1
> 
> but don't be shy, your direct contributions to MDAL are welcomed too!

good call. in retrospective a "missed opportunity". I'll try to find something else to PR.